templates/webapi/test/overview.html.ep

```diff
@@ -217,24 +221,43 @@
 <div class="modal fade" id="add-comments-modal">
     <div class="modal-dialog modal-lg">
         <div class="modal-content">
-            <form action="<%= url_for('apiv1_post_comments')->query(job_id => $job_ids) %>" method="post" onsubmit="addComments(this); return false;">
+            <form method="post">
                 <div class="modal-header">
                     <h4 class="modal-title">Add comment on all currently shown jobs</h4>
                     <button type="button" class="btn-close" data-bs-dismiss="modal"></button>
                 </div>
                 <div class="modal-body">
                     %= include 'comments/add_comment_form_groups', group_comment => 0, nosubmit => 1
                 </div>
+		    
                 <div class="modal-footer">
                     <div id="add-comments-progress-indication">
-                        <div class="flex-fill"><i class="fa fa-cog fa-spin fa-fw"></i> Saving…</div>
+                        
+			<div class="flex-fill"><i class="fa fa-cog fa-spin
+							 fa-fw"></i> Saving…</div>
+			<div class="progress" id="restart-progress-bar">
```

Comment:
This ID is not used. But is that right? Wouldn't one want to set the progress bar value at some point? It looks like this code is actually just a leftover. I suggest to stick with a simpler spinning loading indication anyway.